internal/grpc/grpclog/grpclog_test.go

```diff
@@ -0,0 +1,102 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+
+// grpclog provides a logging utilities for gRPC.
+package grpclog
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"google.golang.org/grpc/connectivity"
+)
+
+func TestLogStateChanges(t *testing.T) {
+	testCases := []struct {
+		name    string
+		conn    getStater
+		isReady bool
```

Comment:
```suggestion
		wantIsReadyCalled bool
```
nit: use want prefix for values you expect, so it is clear which of the fields are input and which expected output.